src/shogun/regression/svr/LibLinearRegression.cpp

```diff
@@ -43,11 +45,19 @@ void CLibLinearRegression::init_defaults()
 
 void CLibLinearRegression::register_parameters()
 {
-	SG_ADD(&m_C, "m_C", "regularization constant",ParameterProperties::HYPER);
-	SG_ADD(&m_epsilon, "m_epsilon", "tolerance epsilon");
-	SG_ADD(&m_epsilon, "m_tube_epsilon", "svr tube epsilon",ParameterProperties::HYPER);
-	SG_ADD(&m_max_iter, "m_max_iter", "max number of iterations");
-	SG_ADD(&m_use_bias, "m_use_bias", "indicates whether bias should be used");
+	SG_ADD(&m_C, "C", "regularization constant", ParameterProperties::HYPER);
+	SG_ADD(
+		&m_epsilon, "m_tube_epsilon", "svr tube epsilon",
+		ParameterProperties::HYPER);
+	SG_ADD(&m_max_iter, "max_iter", "max number of iterations");
+	SG_ADD(&m_use_bias, "use_bias", "indicates whether bias should be used");
+	SG_ADD(
+		(machine_int_t*)&m_liblinear_regression_type,
+		"liblinear_regression_type", "Type of LibLinear regression.");
+	m_enum_to_string_map[get_name()]["liblinear_regression_type"] = {
+		{"L2R_L2LOSS_SVR", 0},
```

Comment:
Use the enum value here rather than the int to make it readable, and they are available here